### PR TITLE
remove duplicate import

### PR DESCRIPTION
**Pull request by Sean-Holcomb**
## Does this PR rely on any other PRs?

- 
- 

## Links to Issues or ZD tickets this PR addresses or fixes

- 
-

### CODE REVIEW

File: pkg/kubecost/window.go [hunk 1]
```diff
@@ -12,7 +12,6 @@ import (
 
 	"github.com/kubecost/cost-model/pkg/env"
 	"github.com/kubecost/cost-model/pkg/thanos"
-	"github.com/kubecost/cost-model/pkg/util/timeutil"
```
Comment (Sean-Holcomb) [inline pkg/kubecost/window.go:15]: This is imported on line 11

**Comment by michaelmdresser** [comment]
Looks like the build is still failing on this, probably related to a recently-merged PR?

**Comment by Sean-Holcomb** [comment]
Looks like there is actually another failing test going to just add that to this PR

**Review by nikovacevic — APPROVED** [review]
Thanks, Sean. This is a good reminder to check that the build still passes after merging PRs.

**Pull request merged**